### ISSUE TITLE
[Fiche espèce] Représentation graphique des répartitions altitudinales et de phénologie

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature#2252]
Il serait intéressant de pouvoir visualiser des éléments de répartition de la phénologie et d'altitude de l'espèce dans la fiche taxon du module synthese, telle qu'elle est représentée dans l'atlas, peut-être sur unité plus fine (décade par exemple). Notamment pour mieux cerner le cohérence de la donnée dans le module validation.

![image](https://user-images.githubusercontent.com/22891423/211359524-f8dc1f52-45a5-4bbf-aa6d-36d90cd6c0bb.png)

![image](https://user-images.githubusercontent.com/22891423/211359398-cdc3aca9-1f9b-4275-9ef5-9114c3f3009a.png)

**Comment by DonovanMaillard** [comment]
Oui, c'était envisagé dans un premier temps mais pas eu le temps de le mettre en place. 

Pour un aspect analytique, j'aurais même aimé personnellement qu'on ait un grahpique avec la phénologie en X, les altitudes en Y, et une "enveloppe" qui nous donne les altitudes min/max des données valides sur chaque période, puisqu'on peut avoir plusieurs générations en basse altitude et pas en haute par exemple. Mais pas forcément simple à mettre en place...

**Comment by JeromeMaruejouls** [comment]
Bonjour à tous,
Sur https://biodiv-occitanie.fr/ , j'ai mis en place des graphiques par décades, et par zones biogéographiques. Comme précisé à @camillemonchicourt , je vais le faire remonter à la communauté pour intégration à une prochaine version de l'atlas. Hésitez pas à faire vos remarques, suggestions (le code n'est pas très propre pour le moment, mais ça va être l'occasion de le nettoyer).
